CMake/Servus.cmake

```diff
@@ -0,0 +1,14 @@
+
+set(SERVUS_PACKAGE_VERSION 0.1)
+set(SERVUS_REPO_URL https://github.com/HBPVis/servus.git)
+set(SERVUS_DEPENDS OPTIONAL DNSSD avahi-client Boost)
+set(SERVUS_BOOST_COMPONENTS "unit_test_framework")
+set(SERVUS_DEB_DEPENDS libboost-test-dev avahi-daemon libavahi-client-dev)
+
+if(CI_BUILD_COMMIT)
```

Comment:
I guess it's part of the config file Servus.cmake (PR for this?). I vote for removing it, and then this is a +1 here, so we get this merged soon.